### DISCUSSION TITLE
Data source connection error

### DISCUSSION THREAD

**Discussion by krasotinpa** [netbox-community/netbox#16463] [General]
Hi all - 

I'm trying to connect  our local gitlab as a data source of scripts, authentication is by login/password.
Data source is configured and enabled, but Sync causes error:
`UnboundLocalError("cannot access local variable '_' where it is not associated with a value")`

`git clone https://gitlab....` works fine on the NETBOX host.

Tell me please what could be the problem?

NETBOX version is v3.7.8.

**Comment by markkuleinio** [comment]
https://github.com/netbox-community/netbox/issues/15510

**Comment by krasotinpa** [comment]
Thank you [Markku Leiniö](https://github.com/markkuleinio)!
